test/libsolidity/semanticTests/revertStrings/empty_v1_yul.sol

```diff
@@ -1,18 +0,0 @@
-contract C {
```

Comment:
This test is a duplicate of `empty_v1.sol` with the weird set of conditions: ABIEncoderV1Only: true andcompileViaYul: true